extract_pipeline.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import pandas as pd
+
+response = requests.get('https://randomuser.me/api/')
+
+data = response.json()
+results = data['results']
+
+normalized_result = pd.json_normalize(results)
```

Comment:
This can be a module also, a module that normalise a json based on pandas DF

Reply:
Now on it